### DISCUSSION TITLE
After using "npm run dev" command a lot of weird commands are displayed

### DISCUSSION THREAD

**Discussion by DimaKozhanovsky** [open-tutorials/cypress#66] [Q&A]
Colleagues, I'm doing a tutorial "Deploy project Conduit locally" fourth paragraph https://www.epic1h.com/build_conduit.

My expectations, this is a possibilite to  use project on http://localhost:3000/ .

Nevertheless  i after using `npm run dev `command a lot of weird commands  in terminal are displayed.

Please , help me to find out the reason of issue ? Thank you in advance 🙏

<img width="1757" alt="Screen Shot 2023-02-21 at 22 11 39" src="https://user-images.githubusercontent.com/74324758/220437240-f0edc1b2-7ccc-4f10-adff-895463b8239a.png">

**Comment by Vladimir-Tarasov** [comment]
Good day. Will check.

**Reply by DimaKozhanovsky** [reply]
Сan't say for sure , i forgot , but in whole the reason was that backend and frontend should have been run separately , we were discussing it at stand up , i will re watch and probably find the right answer @Vladimir-Tarasov @breslavsky
